### PR TITLE
Add a makefile for importing the source code in DVT Eclipse IDE

### PR DESCRIPTION
**Pull request by gabrielrad**
This pull request allows user to automatically import the core-v-verif project in [DVT Eclipse IDE](https://www.dvteclipse.com/products/dvt-eclipse-ide).

@MikeOpenHWGroup I'm asking you to review this, because you might be more familiar than others about DVT. Thank you!

Signed-off-by: gabriel.raducan <gabriel.raducan@amiq.com>

### CODE REVIEW

File: mk/README.md
Comment (MikeOpenHWGroup) [inline mk/README.md:5]: I'm not sure this is correct.   In core-v-verif (and indeed, within most of the RISC-V  community), the term "toolchain" is typicalled used to mean the cross-compiler used to compile/assemble either C or RISC-V assembler source code into RISC-V machine code.   I doubt that we need to use the toolchain to import DVT.

File: mk/uvmt/dsim.mk
Comment (MikeOpenHWGroup) [inline mk/uvmt/dsim.mk:349]: I see that there is a similar code segment in each of the simulator-specified Makefiles.  It would ease maintenance considerably if this code could be placed in Common.mk.

File: mk/uvmt/dvt.mk
Comment (MikeOpenHWGroup) [inline mk/uvmt/dvt.mk:1]: Nice.   Thanks for using the proper header!

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
Thanks very much for this contribution @gabrielrad.  As you can see, I have made a few comments that we should address before merging in the code.  Minor stuff really, and probably the quickest way to do it is over a short Zoom call.

Also, and this is _very_ important, I cannot merge this pull request until it is covered by the appropriate Contributor Agreement.   This is how the OpenHW Group ensures our open-source code is, in fact, legally open-source.   I will reach out to the AMIQ EDA team about that via email.

**Review by MikeOpenHWGroup — APPROVED** [review]
Thanks for the quick fixes @gabrielrad.   I'll merge this in as soon as the Eclipsefdn/eca is completed.

**Comment by gabrielrad** [comment]
Thank you for the review @MikeOpenHWGroup !

**Review by MikeOpenHWGroup — APPROVED** [review]
Looks good @gabrielrad. I'm running a CI regression just to be doubly sure, but as this PR does not affect existing code, that is just me be hyper-cautious.   Will merge this in when the CI regression completes.

**Comment by ruspl-afed** [comment]
@gabrielrad Do you plan to publish an example of the project that is created by IDE as a result of the import?

**Comment by MikeOpenHWGroup** [comment]
Hi @ruspl-afed, I will be meeting with @gabrielrad later this week to discuss next steps.  Would you like to be in that meeting?

**Comment by gabrielrad** [comment]
@ruspl-afed We might, we're thinking to add a short demo movie on our youtube channel [DVT Eclipse](https://www.youtube.com/user/dvteclipse), but we are open to other suggestions.

**Comment by ruspl-afed** [comment]
> Hi @ruspl-afed, I will be meeting with @gabrielrad later this week to discuss next steps. Would you like to be in that meeting?

Hi @MikeOpenHWGroup , thank you for suggestion. I'm always interested to learn something new in IDE domain.

**Pull request merged**
